Copy Edit troubleshooting.rst (#2609)
ManimCommunity/manim: docs/source/installation/troubleshooting.rst

   pip uninstall <package>

If you cloned manim from GitHub, you should either remove it
or run manim outside that folder.

Other errors
************

``pip install manim`` fails when installing manimpango?
-------------------------------------------------------

> This mostly means that pip was not able to use our pre-built wheels

Reviewer: ```suggestion
This most likely means that pip was not able to use our pre-built wheels
```

of the corresponding `GitHub repository <https://github.com/ManimCommunity/ManimPango>`_.

(Windows) OSError: dlopen() failed to load a library: pango

> ------------------------------------------------------------

Reviewer 2: The underline below a section header should always have the same length as the title itself.

```suggestion
-----------------------------------------------------------
```

This should be fixed in Manim's latest version, update
using ``pip install --upgrade manim``.

> Some letters are missing from Text/Tex output
> ------------------------------------------------------------

Reviewer 2: ```suggestion
Some letters are missing from Text/Tex output
---------------------------------------------
```

.. _dvisvgm-troubleshoot:

Installation does not support converting PDF to SVG

> ----------------------------------------------------

Reviewer 2: ```suggestion
---------------------------------------------------
```

  dvisvgm -V1

> (while still having ``LIBGS`` set to the correct path, of course.) If ``dvisvgm``

Reviewer 2: Not sure I agree with the parentheses here, but that is more of a personal remark. No change required.

--------------------------------

If ``choco install manimce`` failed,

> it is because Python was not added to your ``PATH`` variable properly.

Reviewer 2: This does change the meaning somewhat, and I'm not sure the statement is still true after the change. This should be updated to reflect that if ``choco install manimce`` it does not **always** have to be this particular problem.

Reviewer 3: I think this part of troubleshooting can be removed. In the recent version the installation script searches the windows registry for python and doesn't depend on having it in the PATH variable.

Author: I'll go ahead and remove this line if it's superfluous, then.

Author: Hm... a large section (lines 188t to least 213) refers to the PATH variable... which should I remove and which should I keep? For right now, I'll just update it to say likely.

Try running the following commands in your terminal:

1. ``py --version``
2. ``python --version``
3. ``py3 --version``
4. ``python3 --version``

> Ideally, ``py --version`` and ``python --version`` should return a version.

Reviewer 2: I think "minimally" here is used as "out of these four commands, at least the specified two should return a version, but the others could do so as well". Ideally sort of is a stronger, not necessarily intended meaning, no? Do you have any thoughts about that?

Author: Ah, that makes sense. I agree!
